Account/SyncAuthState.swift

```diff
@@ -68,6 +69,16 @@ open class FirefoxAccountSyncAuthState: SyncAuthState {
         }
     }
 
+    var _clientName: String?
+    public var clientName: String? {
```

Comment:
There's no need for a getter and setter here. Just `public var clientName: String?`.